### ISSUE TITLE
City ID in middle click

### ISSUE THREAD

**Issue by jwrober** [longturn/freeciv21#2211]
**Is your feature request related to a problem? Please describe.**
When evaluating what enemy cities to capture, it would be good to know the ID of the city to gain a sense of the age of the city. Older cities, when captured, give the most territory to the victor.

**Describe the solution you'd like**
Add the ID of the city to the middle click pop up menu.

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by hugomflavio** [comment]
Would it be possible to show the founding date instead? Using in-game time. That would be a neat addition, perhaps nicer than IDs.

**Comment by lmoureaux** [comment]
> Would it be possible to show the founding date instead? Using in-game time. That would be a neat addition, perhaps nicer than IDs.

I don't think we have this information. At least I couldn't find it in the city struct

**Comment by hugomflavio** [comment]
What about the turn when the city was built? If that is stored, perhaps it could be calculated from there.

**Comment by lmoureaux** [comment]
This is what we have:
https://github.com/longturn/freeciv21/blob/ad20326066bb6c1ffef303b475620508b1a88c0a/common/city.h#L283-L454

**Comment by hugomflavio** [comment]
Turn founded: https://github.com/longturn/freeciv21/blob/ad20326066bb6c1ffef303b475620508b1a88c0a/common/city.h#L349

**Comment by lmoureaux** [comment]
I must be blind :D

**Comment by hugomflavio** [comment]
can we back-calculate the year from the turn to make it look fancier?

**Comment by lmoureaux** [comment]
mmmh, I'm not entirely sure we can. Will need to check

**Comment by hugomflavio** [comment]
The turn works well enough for now. I'll try to figure this out once we're done with #2232

**Comment by lmoureaux** [comment]
@blabber posted in another issue:

> I have a commit (https://github.com/longturn/freeciv21/commit/415d813cf5f4406306064063c9dea9afc5071a1f) in my local, that displays the turn_founded for a city in the popup info. Testing shows, that you can rely on turn_founded only for your own cities.
>
> For foreign cities, after reloading a saved game, turn_founded seems to contain the turn number when the game was reloaded. Even before reloading I had the impression, that turn_founded displayed the turn number when I discovered the city rather than when it was founded by the enemy, but I didn't check this any further.
